docs/protocol-methods.rst

```diff
@@ -160,12 +160,19 @@ be confirmed within a certain number of blocks.
 
 **Signature**
 
-  .. function:: blockchain.estimatefee(number)
+  .. function:: blockchain.estimatefee(number, mode=None)
+  .. versionchanged:: 1.5
+     *mode* argument added
 
   *number*
 
     The number of blocks to target for confirmation.
 
+  *mode*
+
+    A string to pass to the bitcoind *estimatesmartfee* RPC as the
+    *estimate_mode* parameter. Optional.
```

Comment:
It's optional, so what is the default?

Reply:
The electrum server is just forwarding these requests to bitcoind. If it is not provided by the client, the server will not pass it to bitcoind either. It is up to bitcoind to choose a default. I guess this could be mentioned here though.

Reply:
done